Sources/Helpers/Factories/ViewCellItemsFactory.swift

```diff
@@ -0,0 +1,160 @@
+//
+//  Copyright © 2020 Rosberry. All rights reserved.
+//
+
+import UIKit
+
+open class ViewCellItemsFactory<Object: GenericDiffItem, View: UIView> {
+
+    public typealias Cell = CollectionViewViewCell<View>
+    public typealias CellItem = CollectionViewViewCellItem<Object, View>
+    typealias AssociatedFactory = AssociatedCellItemFactory<Object, Cell>
+
+    /// Set this handler to retrieve a specific set of cell items for the associated object
+    ///
+    /// - Parameters:
+    ///    - Object: the object associated with a cell item
+    public var initializationHandler: ((Object) -> [CollectionViewCellItem?])?
```

Comment:
Not sure if documentation for this method is correct 🤔 Retrieve means get, but here we should create cell items for object. Am I wrong?

Reply:
improved to `to retrieve or create`. This handler is removed in following PR (Factories upgrade)